### PR TITLE
[DOC] Restructures Dashboards TOC

### LINKED ISSUE #2621: [DOC] Dashboards TOC
**What do you want to do?**
 
- [X] Request a change to existing documentation
- [ ] Add new documentation
- [ ] Report a technical problem with the documentation
- [ ] Other 

**Tell us about your request.** Provide a summary of the request and all versions that are affected.
 Arrange Dashboards documentation navigation into logical order 

**What other resources are available?** Provide links to related issues, POCs, steps for testing, etc.

### PR DESCRIPTION
**Pull request by vagimeli**
Signed-off-by: vagimeli <vagimeli@amazon.com>

### Description
Restructures Dashboards TOC 

### Issues Resolved
#2621 

### Checklist
- [X] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by vagimeli** [comment]
Proposed TOC structure

Screen Shot 2023-02-03 at 1 13 21 PM

File: _dashboards/quickstart-dashboards.md [hunk 1]
```diff
@@ -11,9 +11,6 @@ redirect_from:
 
 This quickstart guide covers the core concepts that you need to understand to get started with OpenSearch Dashboards. You'll learn how to:
 
-- Add sample data.
-- Explore and inspect data with **Discover**.
-- Visualize data with **Dashboard**.
 - Add sample data.
 - Explore and inspect data with **Discover**.
```
Comment (hdhalter) [inline _dashboards/quickstart-dashboards.md:15]: What is Discover exactly? Can't we just say Explore and inspect data?

File: _dashboards/quickstart-dashboards.md [hunk 2]
```diff
@@ -11,9 +11,6 @@ redirect_from:
...
 - Visualize data with **Dashboard**.
```
Comment (hdhalter) [inline _dashboards/quickstart-dashboards.md:16]: What is Dashboard? Can we just say Visualize data? I don't think the where is as important as the what and why.

File: _dashboards/visualize/viz-index.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: default
-title: Visualize
-nav_order: 4
+title: Building data visualizations with Visualize
```
Comment (hdhalter) [inline _dashboards/visualize/viz-index.md:3]: Can we drop 'with Visualize'?

**Review by hdhalter — COMMENTED** [review]
Looks good, Melissa! I lean toward dropping any UI terms used in our headings. I assume we want to tie ideas to the UI but I think we can handle that in the procedures.

File: _dashboards/quickstart-dashboards.md (hunk 1 above)
Comment (vagimeli) [reply _dashboards/quickstart-dashboards.md:15]: Discover is the application on the UI.

File: _dashboards/quickstart-dashboards.md (hunk 2 above)
Comment (vagimeli) [reply _dashboards/quickstart-dashboards.md:16]: Dashboard is the application on the UI.

File: _dashboards/visualize/viz-index.md (hunk 3 above)
Comment (vagimeli) [reply _dashboards/visualize/viz-index.md:3]: Visualize is the application on the UI.

Comment (vagimeli) [reply _dashboards/visualize/viz-index.md:3]: Screen Shot 2023-02-03 at 10 03 26 AM

Comment (vagimeli) [reply _dashboards/visualize/viz-index.md:3]: @hdhalter Changes made as suggested.

File: _dashboards/quickstart-dashboards.md (hunk 1 above)
Comment (vagimeli) [reply _dashboards/quickstart-dashboards.md:15]: change made

File: _dashboards/quickstart-dashboards.md (hunk 2 above)
Comment (vagimeli) [reply _dashboards/quickstart-dashboards.md:16]: Change made.

File: _dashboards/visualize/viz-index.md (hunk 3 above)
Comment (vagimeli) [reply _dashboards/visualize/viz-index.md]: TOC updates
Screen Shot 2023-02-03 at 2 58 35 PM

**Comment by vagimeli** [comment]
> 

Thanks! I agree with your suggestion.

File: _dashboards/reporting.md [hunk 4]
```diff
@@ -1,19 +1,19 @@
 ---
 layout: default
-title: Reporting
+title: Creating reports from the Dashboards UI
 nav_order: 70
 ---
 
 
-# Reporting
+# Creating reports from the Dashboards UI
```
Comment (alicejw-aws) [inline _dashboards/reporting.md:8]: This looks good to distinguish it from the new "Creating reports with the Reporting CLI" topic that is in-progress. 
nit . .  .maybe say "with" the Dashboards?

File: _dashboards/visualize/maptiles.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+  - /docs/opensearch-dashboards/maptiles/
+---
+
+{%- comment -%}The `/docs/opensearch-dashboards/maptiles/` redirect is specifically to support the UI links in OpenSearch Dashboards 1.0.0.{%- endcomment -%}
+
+# Configuring a Web Map Service (WMS)
+
+The Open Geospatial Consortium (OGC) Web Map Service (WMS) specification is an international specification for requesting dynamic maps on the web. OpenSearch Dashboards includes default map tiles. For specialized maps, you can configure a WMS on OpenSearch Dashboards following these steps:
+
+1. Open and log in to OpenSearch Dashboards at `https://<host>:<port>`. Alternatively, you can connect to OpenSearch Dashboards by connecting to [https://localhost:5601](https://localhost:5601). The default username and password are `admin`.
```
Comment (alicejw-aws) [inline _dashboards/visualize/maptiles.md:17]: nit - maybe just say

"Log into the OpenSearch Dashboards. . .

**Review by alicejw-aws — APPROVED** [review]
Great work! 

made a couple of minor comments

**Review by cwillum — APPROVED** [review]
LGTM

File: _dashboards/reporting.md (hunk 4 above)
Comment (vagimeli) [reply _dashboards/reporting.md:8]: Accepted.

Comment (vagimeli) [reply _dashboards/reporting.md]: Revised

File: _dashboards/visualize/maptiles.md (hunk 5 above)
Comment (vagimeli) [reply _dashboards/visualize/maptiles.md:17]: Revised

Comment (vagimeli) [reply _dashboards/visualize/maptiles.md:17]: Revised

**Pull request merged**